apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/soeknadoversikt/kommerBarnetTilgode/OversiktKommerBarnetTilgode.tsx

```diff
@@ -33,6 +34,17 @@ export const OversiktKommerBarnetTilgode = ({ kommerBarnetTilgode, redigerbar, s
   const bostedsadresse = søker?.bostedsadresse?.find((adresse) => adresse.aktiv === true)
   const foreldersadresse = forelder?.bostedsadresse?.find((adresse) => adresse.aktiv === true)
 
+  const svarTilStatusIcon = (svar: JaNei | undefined): StatusIconProps => {
+    switch (svar) {
+      case JaNei.JA:
+        return 'success'
+      case JaNei.NEI:
+        return 'error'
+      default:
+        return 'warning'
+    }
+  }
+
```

Comment:
Kanskje greit å ha denne funksjonen utenfor komponenten (evt. i `StatusIcon`-filen?), slik at andre komponenter kan gjenbruke logikken? Den ser i alle fall ganske generell ut for meg når jeg leser koden :)

Reply:
Vurderte det. Kan kanskje flyttes til utils.tsx ved siden av svarTilVurderingsstatus